code.json

```diff
@@ -91,22 +91,22 @@
     "name": "ISIS3",
     "organization": "U.S. Geological Survey",
     "description": "GitHub code repository for the Integrated Software for Imagers and Spectrometers (ISIS)",
-    "version": "8.2.0",
+    "version": "8.3.0",
```

Comment:
Just a heads up that this will immediately break the isis docs website once merged into dev. Might be worth waiting until you have the docs built and pushed to the S3 before pushing these changes.

Reply:
Updated the menu.xsl to include 8.2.0 and 8.3.0. Here's the 8.3.0 docs link: https://isis.astrogeology.usgs.gov/8.3.0/